velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -154,6 +154,26 @@ TEST_F(ProbabilityTest, binomialCDF) {
   VELOX_ASSERT_THROW(
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
+TEST_F(ProbabilityTest, cauchyInverseCDF) {
+  const auto cauchyInvCDF = [&](std::optional<double> median,
+                                std::optional<double> scale,
+                                std::optional<double> probability) {
+    return evaluateOnce<double>(
+        "inverse_cauchy_cdf(c0, c1, c2)", median, scale, probability);
+  };
+  EXPECT_DOUBLE_EQ(0.0, cauchyInvCDF(0.0, 1.0, 0.5).value());
+  EXPECT_DOUBLE_EQ(1.0, cauchyInvCDF(0.0, 1.0, 0.75).value());
```

Comment:
Please add test cases with kDoubleMax, kDoubleMin, kInf, kNan for each parameter.